Gulpfile.js

```diff
@@ -98,6 +99,13 @@ gulp.task('standard', () => {
     }))
 })
 
+// Run HTML Hint checks
+gulp.task('html-hint', () => {
+  return gulp.src('./src/views/*.html')
```

Comment:
Having tested this, its ignoring the `/src/views/partials` folder. Stuff in here needs to be included as well, whilst `/src/views/govuk_template_mustache` needs to be ignored.

If this was easy, where would be the fun it!?

Reply:
Oops. I'll have a look! Maybe hold off on reviewing the prototype changes then as this may have a similar issue there.